src/test.cpp

```diff
@@ -36,12 +39,17 @@ void TestSuite::add(Test test) {
     tests[num_tests++] = test;
 }
 
+int main() { // Test entry point
+    return _global_tests.run();
+}
+
 unsigned int TestSuite::run() {
     std::printf("Running %d tests\n", num_tests);
     unsigned int succeeded = 0;
 
     for (unsigned int i = 0; i < num_tests; i++) {
         GameState state = {};
+        _test_gs = &state;
```

Comment:
It's not the nicest solution but it gets the job done. :+1:

Reply:
I envision this getting changed later, IF we decide to multithread the tests